packages/dev-utils/src/viem/chains.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * NOTE: to prevent cyclic imports copied from cli/src/package-to-be/chains.ts
+ * TODO: remove this
```

Comment:
yeah i dont love this being in dev-utils (which so far only has test stuff)

Reply:
easy to solve once package-to-be is an actual package

Reply:
yeah i reverted the deletion and copied the file so we have code duplication but i left the todo